mm/Kconfig

```diff
@@ -98,12 +98,6 @@ config MM_REGIONS
 		that the memory manager must handle and enables the API
 		mm_addregion(heap, start, end);
 
-config MM_MAP_COUNT_MAX
-	int "The maximum number of memory map areas for each task"
-	default 1024
-	---help---
-		The maximum number of memory map areas for each task.
-
```

Comment:
@xiaoxiang781216 could you please elaborate further? Why isn't it necessary to limit the number of map entries? Which other alternative do we have? Please include this information in the commit message!

Reply:
The implementation could support any number map if the system has enough memory, so it's an artificial limitation and better to remove to simplify the configuration.